src/main/java/gamer/JudgeGamer.java

```diff
@@ -0,0 +1,111 @@
+/**
+* @Package_name   : gamer 
+* @Class_name     : JudgeGamer 
+*
+* Create Date : 2023-10-21 
+* Create User : 정은채 
+*/
+package gamer;
+
+import java.util.ArrayList;
+import display.ConsoleDisplay;
+import utility.Const;
+
+public class JudgeGamer implements Gamer{
+
+	// 야구게임 정답 여부
+	public boolean correct;
+	// 기본생성자
+	public JudgeGamer() {	
+		correct = false;
+	} 
+	
+	/**
+	* Description : JudgeGamer의 게임 플레이
+	*
+    * @Method : play
+    */ 
+	@Override
+	public void play() {
+		String result = judgeNum();
+		ConsoleDisplay display = new ConsoleDisplay();
+		display.output(result);
+	}
+	
+	// 컴퓨터의 정답 숫자리스트, 사용자의 예상 숫자리스트
+	private ArrayList<Integer> comNumberArr;
+	private ArrayList<Integer> userNumberArr;
+	/**
+	* Description :  컴퓨터의 정답 숫자리스트와 사용자의 예상 숫자리스트 세팅
+	*
+    * @Method : setNumberArr
+	*/ 
+	public void setNumberArr(ArrayList<Integer> _comNumArr, ArrayList<Integer> _userNumArr) {
```

Comment:
```suggestion
	public void setNumberArr(ArrayList<Integer> comNumArr, ArrayList<Integer> userNumArr) {
```

파라미터도 일관성 있는 이름을 사용하시는 것이 좋습니다. 다른 메서드와 동일하게 언더바(`_`)를 제거해보세요.

Reply:
앗 이름이 같으면 _ 추가하는 거라고 알고 있었는데 아닌가 보군요ㅠㅠ 감사합니다!

Reply:
```java
public void setNumberArr(ArrayList<Integer> comNumArr, ArrayList<Integer> userNumArr) {
  this.comNumArr = comNumArr;
  this.userNumArr = userNumArr;
}
```

파라미터와 클래스 내 멤버 변수의 이름이 같은 경우, 이를 구분하기 위해 `this`를 붙이면 됩니다! 2주차부터 적용해보세요:)